docs/reference/cat/nodes.asciidoc

```diff
@@ -149,9 +155,6 @@ Node uptime, such as `17.3m`.
 `completion.size`, `cs`, `completionSize`::
 Size of completion, such as `0b`.
 
-`dense_vector.value_count`, `dvc`, `denseVectorCount`::
```

Comment:
Note: @jimczi this does not work, so I removed it from the documentation. It can be added back if we decide to fix support for `_cat/nodes`.